samples/SQL/audit-sql-db-without-hybrid-license/azurepolicy.json

```diff
@@ -10,6 +10,10 @@
                 "field": "type",
                 "equals": "Microsoft.SQL/servers/databases"
               },
+              {
+                "field": "kind",
+                "contains": "vcore"

```

Comment:
can you explain this change?

Reply:
Just going after databases wasn't quite selective enough because in order to be eligible for the Hybrid benefit, the database must use the vCore model. The intent of the rule is to provide actionable audits when otherwise eligible databases have not been configured for the hybrid licensing benefit and this change helps reduce the risk of additional noise reducing the effectiveness of the audit data